app/static/css/themes/nord-search-light.css

```diff
@@ -0,0 +1,35 @@
+.autocomplete {
+    position: relative;
+    display: inline-block;
+    width: 100%;
+}
+
+.autocomplete-items {
+    position: absolute;
+    border: 1px solid #685e79;
+    border-bottom: none;
+    border-top: none;
+    z-index: 99;
+
+    /*position the autocomplete items to be the same width as the container:*/
+    top: 100%;
+    left: 0;
+    right: 0;
+}
+
+.autocomplete-items div {
+    padding: 10px;
+    cursor: pointer;
+    color: #3B4252;
+    background-color: #D8DEE9b3;
+    border-bottom: 1px solid #242424;
+}
+
+.autocomplete-items div:hover {
+    background-color: #404040;
+}
+
+.autocomplete-active {
+    background-color: #685e79 !important;
+    color: #f9fafb;
```

Comment:
The `background-color` here clashes a bit with the default color, and it doesn't look like the `color` is getting applied since it isn't flagged as important.

![image](https://user-images.githubusercontent.com/33362396/95664413-9e1f4400-0b15-11eb-9e67-1e1927428a14.png)